### ISSUE TITLE
Improve overall website.

### ISSUE THREAD

**Issue by utk2103** [DevXsphere/Website-devxsphere#16]
- all the description has been provided in this repo.
- you can add figma file or design module for other contributers.
- https://github.com/DevXsphere/Website-devxsphere/tree/main/frontend#readme

**Comment by harsh0728** [comment]
yes I can help in this issue Assign me this issue I will solve this issue as soon as possible

**Comment by MadhavDhatrak** [comment]
I would like to work on this issue. Could you please assign it to me?"

**Comment by utk2103** [comment]
- start working on the project whose work is good i will assign to them.
- try uploading video or photo show casing the work you've done.

**Comment by shivsh01** [comment]
https://github.com/user-attachments/assets/f4519918-2812-461f-a880-42ec962e74b4

Hey @utk2103,

I wanted to let you know that I've made some updates to your website, including changes to the footer, navbar, team section, and team testimonials. Additionally, I've ensured that everything is responsive.
